webservices/templates/swagger-ui.html

```diff
@@ -30,16 +29,102 @@
 </head>
 
 <body>
-    <div id="swagger-ui"></div>
-    <script src="{{ swagger_static('dist/swagger-layout.js') }}"> </script>
+<div id="swagger-ui"></div>
+
+ <script type="text/javascript">
+   /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
+   var apiUmbrellaSignupOptions = {
+     // Pick a short, unique name to identify your site, like 'gsa-auctions'
+     // in this example.
+     registrationSource: 'fec-api',
+
+     // Enter the API key you signed up for and specially configured for this
+     // API key signup embed form.
+     apiKey: "{{ api_key_signup_key }}",
+     // document.write(apiKey.toLocaleString( ));
+     // Provide a URL or e-mail address to be used for customer support.
+     //
+     // The format for e-mail addresses can be given as either
+     // 'example@example.com' or 'mailto:example@example.com'.
+     contactUrl: 'APIinfo@fec.gov',
+
+     // Provide the name of your developer site. This will appear in the
+     // subject of the welcome e-mail as "Your {{siteName}} API key".
+     siteName: 'openFEC',
+
+     // Provide a sender name for who the welcome email appears from. The
+     // actual address will be "noreply@api.data.gov", but this will
+     // change the name of the displayed sender in this fashion:
+     // "{{emailFromName}} <noreply@api.data.gov>".
+     emailFromName: 'openFEC',
```

Comment:
emailFromName still shows as `noreply@api.data.gov`. It should be replaced with fec email address. 
Example:"{{fec.gov}} <noreply@apiinfo@fec.gov>"

Reply:
`emailFromName` shows above as `'openFEC`'. This is not the actual email address, this is the "sender name" that the email actually comes from. If we want a different name from openFEC we can add that instead. But the contact email address is given earlier in the code as `'APIinfo@fec.gov'`. Please let me know if you still think we need to immediately change any values.

Reply:
If user is coming to FEC API to signup for a key then i think we should have FEC email in sender section. 
Screenshot for your reference.

<img width="543" alt="Screen Shot 2020-05-28 at 2 15 18 PM" src="https://user-images.githubusercontent.com/11650355/83178458-88712b00-a0ee-11ea-8c07-70cfa692939b.png">

Reply:
@pkfec thank you! Yes, I totally agree and believe the concern is already being addressed under this comment: https://github.com/fecgov/openFEC/pull/4359#discussion_r431458954. The `emailFromName` is openFEC however. I think the end result will be what you are expecting (just check that other comment)